roles/setup/tasks/main.yml

```diff
@@ -21,8 +21,8 @@
     mode: '0644'
     owner: "{{ receptor_user }}"
     group: "{{ receptor_group }}"
-  # notify:
-  #   - "Receptor Reload"
+  notify:
+    - Restart Receptor service
```

Comment:
This is still supposed to be an attempted reload, not a restart.

Reply:
https://github.com/ansible/automation-platform-collection/blob/devel/roles/receptor/tasks/main.yml#L154

Reply:
:man_facepalming: I'm dumb and changed the wrong task.